draft-packages/form/KaizenDraft/Form/TextAreaField/README.mdx

```diff
@@ -0,0 +1,140 @@
+---
+title: Text Area Field
+navTitle: Text Area Field
+summaryParagraph: A text area includes a label and a longer area you can type multiple lines of text into.
+tags: ["Text area", "Textarea", "Multiline text field", "Textarea Autosize"]
+needToKnow:
+- "For sizing, consider likely text input and use existing data where possible."
+demoStoryId: textareafield-react--default-kaizen-site-demo
+---
+
+import WhenToUseAndWhenNotToUse from "docs-components/WhenToUseAndWhenNotToUse"
+import WhenToUse from "docs-components/WhenToUse"
+import WhenNotToUse from "docs-components/WhenNotToUse"
+
+import DoAndDontContainer from "docs-components/DoAndDontContainer"
+import Do from "docs-components/Do"
+import Dont from "docs-components/Dont"
+
+## Visuals
+
+### UI Kit
+
+<iframe style="border: none;" width="744" height="450" src="https://www.figma.com/embed?embed_host=share&url=https%3A%2F%2Fwww.figma.com%2Ffile%2Fe3TyAOZKfLg9cA01Iy1Mgm%2FKaizen-Site-embed%3Fnode-id%3D50186%253A40775&chrome=DOCUMENTATION" allowfullscreen></iframe>
+
+## To keep in mind
+
+* **Optional vs required:**
```

Comment:
Is this extendable to other form inputs? Should we instead move it to this page: https://cultureamp.design/guidelines/forms/

Reply:
Added a link to the forms page 👍